frontend/src/pages/accounts/profile.module.scss

```diff
@@ -48,31 +48,112 @@
     }
 
     .account-stats {
-      display: flex;
+      display: grid;
+      grid-template-columns: 1fr 1fr;
+      gap: $spacing-md;
       text-align: center;
       justify-content: space-evenly;
 
+      @media screen and #{$mq-md} {
+        display: flex;
+      }
+
       .stat {
+        flex: 1 0 auto;
         display: flex;
         flex-direction: column;
         align-items: center;
         justify-content: space-between;
-        padding: 0 $spacing-md;
-        border-right: 1px solid $color-light-gray-20;
-
-        &:last-child {
-          border-right: 0 none transparent;
+        gap: $spacing-xs;
+        padding: $spacing-md;
+        background-color: $color-white;
+        border: 1px solid $color-light-gray-20;
+        border-radius: $border-radius-md;
+
+        @media screen and #{$mq-lg} {
+          // On small screens, stats have a border and all that, but since
+          // Protocol doesn't have max-width media queries, we need to unset
+          // all those for larger screens:
```

Comment:
Yeah I suppose so (it's pretty much [this larger discussion](https://alistapart.com/article/mobile-first-css-is-it-time-for-a-rethink/)), but since I haven't really interacted with Protocol folks yet, I'm a bit hesitant about barging in and telling them how to do their job... But let's try to keep this in mind for when we're having the Nebula/design system discussion in the work week. (/cc @maxxcrawford - you're putting together the agenda for that, right?)